_summaries/glorot2010understanding_2.md

```diff
@@ -1,19 +1,31 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+paper: glorot2010understanding
+author: elampietti
+score: 8/10
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+The core idea of this paper is to challenge the effectiveness of asymmetric activation functions, such as sigmoid, with the traditional, random initialization for deep architecture.
+The paper accomplishes this by providing improved results from their normalized initialization process and by using symmetric activation functions such as hyperbolic tangent and softsign. 
+
+The Shapeset-3 x 2 dataset is used in this paper for the test models to predict 9 possible shape classifications.
```

Comment:
@elampietti consider briefly explaining the architecture of the model, just so the reader can understand the set up of the experiments that were preformed.